devices/HapticGlove/include/HapticGlove.h

```diff
@@ -0,0 +1,244 @@
+/**
+ * @file HapticGlove.h
+ * @authors  Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef HAPTIC_GLOVE_H
+#define HAPTIC_GLOVE_H
+
+
+#include <Wearable/IWear/IWear.h>
+
+#include <yarp/dev/DeviceDriver.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/SerialInterfaces.h>
+#include <yarp/dev/PreciselyTimed.h>
+#include <yarp/os/PeriodicThread.h>
+
+
+namespace wearable {
+    namespace devices {
+        class HapticGlove;
+    } // namespace devices
+} // namespace wearable
+
+class wearable::devices::HapticGlove :
+        public yarp::dev::DeviceDriver,
+        public yarp::os::PeriodicThread,
+        public yarp::dev::IPreciselyTimed,
+        public wearable::IWear
+{
+private:
+    class SenseGloveImpl;
+    std::unique_ptr<SenseGloveImpl> m_pImpl;
+public:
+    HapticGlove();
+    ~HapticGlove() override;
+
+    // DeviceDriver
+    bool open(yarp::os::Searchable& config) override;
+    bool close() override;
+
+    // PeriodicThread
+    void run() override;
+    void threadRelease() override;
+
+
+    // ================
+    // IPRECISELY TIMED
+    // ================
+
+    yarp::os::Stamp getLastInputStamp() override;
+
+    // =====
+    // IWEAR
+    // =====
+
+    // GENERIC
+    // -------
+
+    WearableName getWearableName() const override;
+
+    WearStatus getStatus() const override;
+
+    TimeStamp getTimeStamp() const override;
+
+    SensorPtr<const sensor::ISensor> getSensor(const sensor::SensorName name) const override;
+
+    VectorOfSensorPtr<const sensor::ISensor> getSensors(const sensor::SensorType) const override;
+
+    ElementPtr<const actuator::IActuator> getActuator(const actuator::ActuatorName name) const override;
+
+    VectorOfElementPtr<const actuator::IActuator> getActuators(const actuator::ActuatorType type) const override;
```

Comment:
@kouroshD this line stills seems too long to me, did you process this file with clang format?

Reply:
I have made a dummy change in the file, and saved it. So, now it should be consistent with the clang format.